### PR TITLE
docs: update Data Visualizer docs for 2.35

### PR DESCRIPTION
**Pull request by edoardo**
DV 2.35 updates to the docs

- [x] Update vis type selector with Stacked Area [DHIS2-7855](https://jira.dhis2.org/browse/DHIS2-7855) and [DHIS2-9217](https://jira.dhis2.org/browse/DHIS2-9217).
- [x] New screenshot for Period selector [DHIS2-8807](https://jira.dhis2.org/browse/DHIS2-8807)
- [x] New screenshot for Data selector
- [x] Option dialog - Series tab. Multi type, 4 multi axis. Plus remove the old multiple axes info. Plus info about which other options are disabled when using multi axes. [DHIS2-6385](https://jira.dhis2.org/browse/DHIS2-6385)
- [x] Text styling options [DHIS2-8426](https://jira.dhis2.org/browse/DHIS2-8426)
- [x] Color sets [DHIS2-670](https://jira.dhis2.org/browse/DHIS2-670)
- [x] Update legend key title [DHIS2-9003](https://jira.dhis2.org/browse/DHIS2-9003)
- [x] Section about two category charts [DHIS-7876](https://jira.dhis2.org/browse/DHIS2-7876)

### CODE REVIEW

File: src/commonmark/en/content/user/data-visualizer.md [hunk 1]
```diff
@@ -188,6 +196,16 @@ The organisation units dialog is flexible, offering essentially three ways of se
...
 ![](resources/images/data-visualizer/data-visualizer-organisation-unit-dimension-modal.png)
 
+### Two category charts
+
+Most chart visualization types can show two categories.
+Switching from Pivot Table to Column, Bar, Area (and their stacked versions) and Line is keeping the first two dimensions in Category, any additional dimension is moved to Filter.
+The labels for the first dimension in Category are shown at the top of the chart, and the ones for the second dimension at the bottom.
+The resulting visualization is composed of separate charts, one for each item in the first dimension.
+
+Example screenshot:
```
Comment (martinkrulltott) [inline src/commonmark/en/content/user/data-visualizer.md]: I would remove this text (_Example screenshot:_) as we don't have texts like this for the other screenshots.

File: src/commonmark/en/content/user/data-visualizer.md [hunk 2]
```diff
@@ -438,15 +467,17 @@ Assigned Categories is a composite dimension that represents associated category
 
 Another way of adding assigned categories is by accessing the **Add Assigned Categories** option from the `Data` dimension's context menu (not available for `Gauge`, `Year over year` or `Single value`).
 
-## Adding more axes
+## Adding more axes and changing how different series are displayed
```
Comment (martinkrulltott) [inline src/commonmark/en/content/user/data-visualizer.md]: For consistency in the **List of available options**, maybe we can add a **Series tab** section with a reference to the the **Adding more axes** section. 
E.g. 

> **Series tab**
> _Options for adding more axes and changing how different series are displayed are set in this section. Please see a detailed description of how this works in the corresponding section below._

File: src/commonmark/en/content/user/data-visualizer.md (hunk 1 above)
Comment (edoardo) [reply src/commonmark/en/content/user/data-visualizer.md]: Removed.

File: src/commonmark/en/content/user/data-visualizer.md (hunk 2 above)
Comment (edoardo) [reply src/commonmark/en/content/user/data-visualizer.md]: Done.

File: src/commonmark/en/content/user/data-visualizer.md (hunk 1 above)
Comment (martinkrulltott) [reply src/commonmark/en/content/user/data-visualizer.md]: Fixed

File: src/commonmark/en/content/user/data-visualizer.md (hunk 2 above)
Comment (martinkrulltott) [reply src/commonmark/en/content/user/data-visualizer.md]: Fixed

**Pull request merged**